src/actions/guides/database/models.cr

```diff
@@ -54,29 +54,46 @@ class Guides::Database::Models < GuideAction
     Your model will inherit from `BaseModel` which is just an abstract class. You can use
     this to define methods all of your models should have access to.
     Next you'll see the `table` block that defines which table this model is connected to.
-    If your table defined the primary key column as `UUID`, you'll need to add the `primary_key_type`
-    option here as well.
-
-    ```crystal
-    # src/models/user.cr
-    class User < BaseModel
-      # Sets the type for `id` to `UUID`
-      table :users, primary_key_type: :uuid do
-      end
-    end
-    ```
 
     ## Defining a column
 
     ### Default columns
 
     By default, Lucky will add a few columns to your model for you.
 
-    * `id` - Your primary key column. Default `Int32` unless you specified `UUID`.
+    * `id` - Your primary key column. Default `Int64`
     * `created_at` - default `Time` type.
     * `updated_at` - default `Time` type.
 
-    > These columns are autogenerated and managed by Avram for you.
+    To skip adding these, call the `skip_default_columns` macro in your model or `BaseModel`.
```

Comment:
I think we should first show how to change the default columns. To do that you add this to your model:

```crystal
macro default_columns
  primary_key custom_id_name : UUID
  timestamps
end
```

That  way you don't need to add them for every model.

`skip_default_columns` is most likely only going to  be used when you have like 1 or 2 where you don't have timestamps or something